src/deepsparse/loggers/metric_functions/helpers.py

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Any
+
+
+__all__ = ["apply_function"]
+
+
+def apply_function(value: Any, function: str):
+    if function not in ["identity_1", "identity_2"]:
+        raise NotImplementedError()
+    else:
+        return value
```

Comment:
also the function wasn't applied? should it?

Reply:
Please see the current state, should be less confusing